docs/source/getting_started/command_line_tool.rst

```diff
@@ -155,6 +155,9 @@ Once you have a trained model checkpoint, you can create an embedding of a datas
     # use custom checkpoint
     lightly-embed input_dir=cat checkpoint=mycheckpoint.ckpt
 
+    # use the last checkpoint you created
+    lightly-embed input_dir=cat checkpoint=$LIGHTLY_LAST_CHECKPOINT_PATH
```

Comment:
I would mention somewhere in an earlier section with an infobox or so that we use environment variables to save certain information. Just to give the user some general idea of how this variable appears and can be used.
Maybe also add a list of the relevant env variables available.

Reply:
I added this information to the respective commands.